lib/koans/10_map_sets.ex

```diff
@@ -15,8 +15,11 @@ defmodule MapSets do
     assert MapSet.size(new_set) == ___
   end
 
-  koan "Even though I am like a list I am unordered after 32 elements" do
-    assert 1..33 |> MapSet.new() |> Enum.fetch(0) == {___, 11}
+  koan "Even though I am like a list I am unordered, especially with 33+ elements" do
```

Comment:
What about "Event though I seem to act like a list, ordering isn't guaranteed, especially with more than 32 elements"?